WISE-Community/WISE
# Show error page when google id is not registered with WISE

How to test:
1. Try to log in with a google id that has not been registered with WISE before. This should show an error message and a link to join WISE.
2. Make sure that logging in with a google id that has been registered before still works.
3. Make sure that logging in with username/password (not using google) still works.

#1419
